packages/terra-overlay/package.json

```diff
@@ -32,9 +32,10 @@
     "classnames": "^2.2.5",
     "focus-trap-react": "^4.0.1",
     "prop-types": "^15.5.8",
-    "terra-button": "^2.28.0",
-    "terra-doc-template": "^1.19.0",
-    "terra-icon": "^2.26.0"
+    "react-portal": "^4.1.2",
+    "terra-button": "^2.27.0",
+    "terra-doc-template": "^1.18.0",
+    "terra-icon": "^2.25.0"
```

Comment:
Whoops, button, doc-template and icon versions need bumped to match latest released versions